models/tmp/stg_hubspot__merged_deal_tmp.sql

```diff
@@ -0,0 +1,4 @@
+{{ config(enabled=fivetran_utils.enabled_vars(['hubspot_sales_enabled','hubspot_deal_enabled'])) }}
```

Comment:
Same comment about possibly adding a new variable here for the new table. If it is low enough then we should create a new variable for this model. I could see it being common that not enough accounts leverage merged deals in HubSpot and we should create a new variable.

Reply:
added here as well